netbox/dcim/tests/test_models.py

```diff
@@ -512,6 +512,11 @@ def setUpTestData(cls):
                 FrontPort(device=patch_panel, name='Front Port 4', rear_port=rearport, rear_port_position=4, type=PortTypeChoices.TYPE_8P8C),
             ))
 
+        # Create a 1-on-1 patch panel
+        patch_panel = Device.objects.create(device_type=devicetype, device_role=devicerole, name='Panel 5', site=site)
+        rearport = RearPort.objects.create(device=patch_panel, name='Rear Port 1', positions=1, type=PortTypeChoices.TYPE_8P8C)
+        FrontPort.objects.create(device=patch_panel, name='Front Port 1', rear_port=rearport, rear_port_position=1, type=PortTypeChoices.TYPE_8P8C)
```

Comment:
IMO it would be cleaner to do the panel creation with the others on line 499, then modify the for loop to only act on the first four. That way it's more clear that you're creating five patch panels, four with multiposition rear ports and one with a single position rear port.

It also ensures that the last patch panel is included in the set should we need to assign `patch_panels` to the instance for reference by a test method.